src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -106,4 +109,31 @@ public static MedicalCondition parseMedicalCondition(String medicalCondition) th
         }
         return medicalConditionSet;
     }
+
+    /**
+     * Parses {@code Collection<String> feedTimes} into a {@code Set<FeedTime>}.
+     */
+    public static Set<FeedTime> parseFeedTimes(Collection<String> feedTimes) throws ParseException {
+        requireNonNull(feedTimes);
+        final Set<FeedTime> feedTimeSet = new TreeSet<>(new FeedTimeComparator());
+        for (String feedTimeText : feedTimes) {
+            feedTimeSet.add(parseFeedTime(feedTimeText));
+        }
+        return feedTimeSet;
+    }
+
+    /**
+     * Parses a {@code String feedTimeText} into a {@code FeedTime}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code feedTimeText} is invalid.
+     */
+    private static FeedTime parseFeedTime(String feedTimeText) throws ParseException {
```

Comment:
Unsure if this method should be public (to standardise with the medical conditions methods)

Reply:
Update to public